src/diffr_lib/mod.rs

```diff
@@ -648,9 +649,8 @@ fn to_usize(input: isize) -> usize {
         input as _
     }
 }
-
 #[derive(PartialEq, Eq, Clone, Copy, Debug)]
-enum TokenKind {
+enum GraphemeKind {
```

Comment:
nitpick: let's keep the old name (even if less accurate) since the term "token" is pervasive in the code.